devmgmtui/src/components/VerifyAuthentication.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+export default function(ComposedComponent) {
+    class Authentication extends Component {
+        static contextTypes = {
```

Comment:
what is this used for ?

Reply:
It is a class level property which provides any piece of code in the application the ability to reference _Authentication.contextTypes_.
The HOC still works even if we eliminate this piece of code and go ahead with _this.props.history.push('/')_

Reply:
remove it if it's not required, I do not see any use of it.

Reply:
Alrighty.